spec/discourse_akismet/check_spam_user_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'rails_helper'
+
+describe DiscourseAkismet::CheckSpamUser do
+  before do
+    SiteSetting.akismet_api_key = 'not_a_real_key'
+    SiteSetting.akismet_enabled = true
+
+    user_profile = user.user_profile
+    user_profile.bio_raw = random_bio
+    user_profile.save
+  end
+
+  let(:user) { Fabricate(:newuser) }
+
+  let(:random_bio) { 'random profile' }
```

Comment:
```suggestion
  let(:bio) { 'random profile' }
```
I would just name it `:bio` since there is nothing random about the bio.